content/tutorial-svd.md

```diff
@@ -202,9 +202,9 @@ results in a `ValueError`. This happens because having a one-dimensional array f
 
 ```{code-cell} ipython3
 import numpy as np
-Sigma = np.zeros((768, 1024))
-for i in range(768):
-    Sigma[i, i] = s[i]
+Sigma = np.zeros((U.shape[0], Vt.shape[0]))
+diag_indices = (np.arange(U.shape[0]),) * 2
+Sigma[diag_indices] = s
```

Comment:
Good points, thank you. Also I should mention that I used the `shape` attributes instead of the numeric values as I thought it would make it more likely to work for user-supplied images. However, I certainly see the value of sticking with the numerical values to better illustrate the particular example image, so feel free to switch back to numbers if you think that's more appropriate.